src/app/app.component.ts

```diff
@@ -7,6 +10,43 @@ import {ChangeDetectionStrategy, Component} from '@angular/core';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class AppComponent {
-  date = new Date();
-  number = 1000000;
+  menuItems: MenuItem[] = [
+    {
+      label: 'Dashboard',
+      icon: 'pi pi-home',
+      routerLink: []
+    },
+    {
+      label: 'Reports',
+      icon: 'pi pi-chart-line',
+      items: [
+        {
+          label: 'Report 1',
+          icon: 'pi pi-chart-line',
+          routerLink: []
+        }
+      ]
+    }
+  ];
+
+  profileItems: MenuItem[] = [
+    {
+      label: 'Profile',
```

Comment:
i18n explicitly left out for now I assume?

Reply:
Yes, your assumption is correct.